core/config/envvar/schema.go

```diff
@@ -126,7 +126,7 @@ type ConfigSchema struct {
 	SolanaNodes     string `env:"SOLANA_NODES"`
 	TerraEnabled    bool   `env:"TERRA_ENABLED" default:"false"`
 	TerraNodes      string `env:"TERRA_NODES"`
-	StarkNetEnabled bool   `env:"STARKNET_ENABLED" default:"false"`
+	StarknetEnabled bool   `env:"STARKNET_ENABLED" default:"false"`
```

Comment:
```suggestion
	StarkNetEnabled bool   `env:"STARKNET_ENABLED" default:"false"`
```

How come `StarkNetNodes` is not defined here?

While both `StarkNetEnabled` & `StarkNetNodes` are defined in `core/config/general_config.go`

Reply:
I hesitated to change this one because I thought I remembered that some code assumed it could convert the casing to undescores, as in `StarkNetEnabled` => `STARK_NET_ENABLED`. I could be mistaken though, and this is certainly better if it works. Just something to look for if you have issues later. :shrug:

Reply:
:shrug: They call themselves `StarkWare`/`StarkNet`

Reply:
Let's use `StarkNet`, but `STARKNET` for env vars/underscore case